homeassistant/components/camera/__init__.py

```diff
@@ -500,3 +541,29 @@ def _write_image(to_file, image_data):
             _write_image, snapshot_file, image)
     except OSError as err:
         _LOGGER.error("Can't write image to file: %s", err)
+
+
+async def async_handle_play_stream_service(camera, service):
+    """Handle play stream services calls."""
+    from homeassistant.components.media_player.const import (
+        ATTR_MEDIA_CONTENT_ID, ATTR_MEDIA_CONTENT_TYPE,
+        SERVICE_PLAY_MEDIA, DOMAIN as DOMAIN_MP)
+    from homeassistant.components.stream import async_request_stream
+    hass = camera.hass
+    try:
+        fmt = service.data.get(ATTR_FORMAT)
```

Comment:
```suggestion
        fmt = service.data[ATTR_FORMAT]
```
Since it has default value